nifi-registry/nifi-registry-core/nifi-registry-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -279,6 +279,22 @@ The default UserGroupProvider is the FileUserGroupProvider, however, you can dev
 
 NOTE: Initial User Identities are only created if the specified Users File is missing or empty during NiFi Registry startup. Changes to the configured Initial Users Identities will not take effect if the Users File is populated.
 
+===== DatabaseUserGroupProvider
+
+If you are using database for storing buckets and flows (like described in the 'Database Properties' section above), you can store the users, groups and policies in the same database either. 
+
+**Note:** This UserGroupProvider is available from version 0.8.0.
+
+
+The DatabaseUserGroupProvider takes the DB-connection information from `nifi-registry.properties` file, so in the `authorizers.xml` file you don't need to give any addition information about the DB (full example of `authorizers.xml` file for using database implementation desribed in `DatabaseAccessPolicyProvider` section).
```

Comment:
Minor nitpick, I suggest "database connection" instead of "DB-connection" and "additional" instead of "addition"